### DISCUSSION TITLE
How to have landscape orientation for certain pages

### DISCUSSION THREAD

**Discussion by selvavm** [quarto-dev/quarto-cli#6104] [Q&A]
I want to convert `qmd` to `docx` with headers and footers. Hence, I am using `reference-doc`. However, I need couple of intermediate pages with landscape orientation. 

I came across a [filter](https://stackoverflow.com/questions/73784720/changing-page-orientation-in-word-using-quarto) but this filter does not include header and footer on the landscape pages. 

The [ooxml](http://officeopenxml.com/WPSectionFooterReference.php) provides a means to add header and footer but it needs an Id. 

Is there a way to include header and footer on the landscape page or know the Id in the filter?

**Comment by mcanouil** [comment]
There are various filter as you found or <https://stackoverflow.com/questions/16173081/is-there-an-option-to-control-output-page-orientation-using-knitr-pander-pand#16236268>.

Usually working with Word is not easy from the command line.
Modifying the LUA filter to work nicely with Word is also not trivial unfortunately.

I don't know enough OpenXML and the reference dock file structure to make the change.

**Reply by selvavm** [reply]
Thanks @mcanouil for responding. It is unfortunate :(

**Reply by mcanouil** [reply]
Related:

- https://github.com/quarto-dev/quarto-cli/discussions/6104
- https://github.com/quarto-dev/quarto-cli/discussions/680
- https://github.com/quarto-dev/quarto-cli/issues/8512

**Comment by edvinsyk** [comment]
Here is an extension that unifies landscape orientation between docx, pdf and typst. It’s not well tested, but has worked for me: https://github.com/edvinsyk/quarto-landscape

**Reply by jjyh** [reply]
that definitely works!  thank you so much, we were almost giving up on typst (due to this & gt tables "indigestion")...this orientation flexibility saves a lot of table-splitting recoding for me

**Reply by mcanouil** [reply]
@edvinsyk thanks for sharing!
If not already done, you might want to submit to the two major listings of extensions I'm aware of for discoverability:
- https://github.com/quarto-dev/quarto-web/tree/main/docs/extensions/listings
- https://github.com/mcanouil/quarto-extensions (http://m.canouil.dev/quarto-extensions/)

(None are "official")

**Comment by Fpadt** [comment]
I just installed the quarto pre-release 1.6.23 and landscape come out-of-the-box, see: [https://github.com/quarto-dev/quarto-cli/pull/10581](https://github.com/quarto-dev/quarto-cli/pull/10581),  
else you can use the extension from which it originates [edvinsyk/quarto-landscape](https://github.com/edvinsyk/quarto-landscape)]

**Reply by mcanouil** [reply]
See https://prerelease.quarto.org/docs/authoring/article-layout.html#landscape-mode

(upon release on https://quarto.org/docs/authoring/article-layout.html#landscape-mode)

**Reply by parametri** [reply]
Makes landscape very convenient to use. Is there a way to put header and page numbers appropriately?
I see header on the right side and page numbers on the left side, instead of top and bottom respectively.

**Reply by mcanouil** [reply]
@parametri please open a new discussion dedicated to your question and follow the guidelines written when opening the discussion, i.e., provide context such as a small reproducible example.
